### ISSUE TITLE
Render tables with text rotated by 90 degrees

### ISSUE THREAD

**Issue by atothakisandor** [phax/ph-pdf-layout#25]
For high density tables it would be useful to be able to place vertically rotated texts for example in header rows.

**Comment by atothakisandor** [comment]
Hello, are there any news with this? :) Thanks, Alex

**Comment by phax** [comment]
Shame on me - unfortunately not yet. Thanks for bringing this again to my attention

**Comment by atothakisandor** [comment]
Thanks! It would be great to have it! :)

**Comment by phax** [comment]
Started doing it... stay tuned - my vacation project :D

**Comment by atothakisandor** [comment]
Thanks! Have a nice vacation :)

**Comment by phax** [comment]
Well, my vacation is long over and I couldn't get it to work easily.
The problem is not just the angle, but also the "turning point", as depicted in below image, showing potential examples for "90 degrees right" rotation:
![grafik](https://github.com/phax/ph-pdf-layout/assets/196219/020aabab-b350-44c9-b7aa-6127756446c5)

**Comment by atothakisandor** [comment]
Wow, thank you, looks nice and useful! Also i can make fancier reports now. I'll give it a try soon. Thanks for your efforts again.

**Comment by atothakisandor** [comment]
Hello, are there any chances, that this feature will be included in future releases? :) (maybe utilizing just one of the enlisted alternatives, like V1). 
We are planning to use this useful tool in future projects, and it would be great to have this feature. 
Thanks! :)

**Comment by phax** [comment]
I totally agree, that this would be nice, but it is very hard to fit it into the existing model. But whatever decision I would take, there would soon be the need for another solution as well. And my proposal doesn't even touch arbitrary angles...

If somebody could come up with an initial PR, it would be appreciated
